pkg/gadgets/run/types/types.go

```diff
@@ -85,10 +94,56 @@ type Printer interface {
 	Logf(severity logger.Level, fmt string, params ...any)
 }
 
+// Type represents the type of a field.
+
```

Comment:
```suggestion
// Kind represents the type of a field.
```
(without empty line)

Or, is this comment at the wrong place?

Reply:
I can't remember, probably it was intended for `Type struct`. Anyway, this comment doesn't add so much information, will remove it.